docs/fsspec.rst

```diff
@@ -95,6 +95,13 @@ Using json keyfile string:
 
 Use `client_user_email` if you are using `delegation of authority`_.
 
+Abusive files
```

Comment:
I would make it Additional Parameters / or Config / or settings and describe both - acknowledge_abuse and  trash_only